common/rand.c

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/internal/enclavelibc.h>
+
+uint64_t _rdrand(void)
+{
+    uint64_t r;
+    __asm__ volatile("rdrand %0\n\t" : "=r"(r));
```

Comment:
Can you work with @sushantstripathi to make sure that this compiles one Windows as well? Otherwise we may have to leave them as separate implementations.

Reply:
MSVC does not support inline assembly for x64 code.